<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UatFrame</class>
 <widget class="QFrame" name="UatFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <widget class="TabnavTreeView" name="uatTreeView">
        <column>
          <property name="text">
            <string notr="true">1</string>
          </property>
          <property name="dragEnabled">
            <bool>false</bool>
          </property>
        </column>
      </widget>
    </item>
    <item>
      <widget class="QLabel" name="hintLabel">
        <property name="styleSheet">
          <string notr="true">QLabel { color: red; }</string>
        </property>
        <property name="text">
          <string/>
        </property>
        <property name="textFormat">
          <set>Qt::RichText</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1">
        <item>
          <widget class="QToolButton" name="newToolButton">
            <property name="toolTip">
              <string>Create a new entry.</string>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="icon">
              <iconset resource="../../image/stock_icons.qrc">
                <normaloff>:/stock/plus-8.png</normaloff>:/stock/plus-8.png
              </iconset>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QToolButton" name="deleteToolButton">
            <property name="enabled">
              <bool>false</bool>
            </property>
            <property name="toolTip">
              <string>Remove this entry.</string>
            </property>
            <property name="icon">
              <iconset resource="../../image/stock_icons.qrc">
                <normaloff>:/stock/minus-8.png</normaloff>:/stock/minus-8.png
              </iconset>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QToolButton" name="copyToolButton">
            <property name="enabled">
              <bool>false</bool>
            </property>
            <property name="toolTip">
              <string>Copy this entry.</string>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="icon">
              <iconset resource="../../image/stock_icons.qrc">
                <normaloff>:/stock/copy-8.png</normaloff>:/stock/copy-8.png
              </iconset>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QToolButton" name="moveUpToolButton">
            <property name="toolTip">
              <string>Move entry up.</string>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="icon">
              <iconset resource="../../image/stock_icons.qrc">
                <normaloff>:/stock/arrow_up.png</normaloff>:/stock/arrow_up.png
              </iconset>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QToolButton" name="moveDownToolButton">
            <property name="toolTip">
              <string>Move entry down.</string>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="icon">
              <iconset resource="../../image/stock_icons.qrc">
                <normaloff>:/stock/arrow_down.png</normaloff>:/stock/arrow_down.png
              </iconset>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QToolButton" name="clearToolButton">
            <property name="toolTip">
              <string>Clear all entries.</string>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="icon">
              <iconset resource="../../image/stock_icons.qrc">
                <normaloff>:/stock/delete_list.png</normaloff>:/stock/delete_list.png
              </iconset>
            </property>
          </widget>
        </item>
        <item>
          <widget class="ElidedLabel" name="pathLabel">
            <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <property name="text">
              <string/>
            </property>
            <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="openExternalLinks">
              <bool>true</bool>
            </property>
          </widget>
        </item>
      </layout>
    </item>
  </layout>
 </widget>
 <customwidgets>
    <customwidget>
      <class>ElidedLabel</class>
      <extends>QLabel</extends>
      <header>widgets/elided_label.h</header>
    </customwidget>
    <customwidget>
      <class>TabnavTreeView</class>
      <extends>QTreeView</extends>
      <header>widgets/tabnav_tree_view.h</header>
    </customwidget>
  </customwidgets>
  <resources>
  <include location="../../image/stock_icons.qrc"/>
 </resources>
 <connections/>
</ui>
